docs/diagrams/UiClassDiagram.puml

```diff
@@ -31,30 +34,38 @@ HiddenOutside ..> Ui
 UiManager .left.|> Ui
 UiManager -down-> "1" MainWindow
 MainWindow *-down->  "1" CommandBox
+MainWindow --> "0..1" HelpWindow
 MainWindow *-down-> "1" ResultDisplay
-MainWindow *-down-> "1" PersonListPanel
+MainWindow *-down-> "1" ApplicationListPanel
 MainWindow *-down-> "1" StatusBarFooter
-MainWindow --> "0..1" HelpWindow
+MainWindow --> "0..1" ReminderWindow
```

Comment:
Hmmm the previous diagram shows 0..1 though? I think we didn't change the implementation of help window right? 😅

Reply:
true and yep I don't think we changed the help window 😅 but maybe we can change to 1 for both if the reasoning makes sense? (in case their original diagram has an error then we end up inheriting that error)